HelpSource/themes/classic.css

```diff
@@ -0,0 +1,36 @@
+:root {
+    /* editor colors */
+    --cm-border-color: #e1e1e1;
+    --cm-background-color: #fff;
+    --cm-selection-color: #308cc6;
+    --cm-keyword-color: #0000e6;
+    --cm-built-in-color: #3333bf;
+    --cm-number-color: #980099;
+    --cm-symbol-color: #007300;
+    --cm-class-color: #0000d2;
+    --cm-primitive-color: #3333bf;
+    --cm-char-color: #007300;
+    --cm-env-var-color: #8c4614;
+    --cm-comment-color: #bf0000;
+    --cm-string-color: #5f5f5f;
+    --cm-text-color: #000;
+
+    /* derived colors
+    in future use `color-mix` to calculate them which allows for fewer variables
+    but currently the qt browser does not support it, see
+    https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix
+    */
+    --sc-bg-color: var(--cm-background-color);
+    --sc-bg-warning: var(--cm-selection-color);
+
+    --sc-text-color: var(--cm-text-color);
+    --sc-text-warning: var(--cm-keyword-color);
+    --sc-text-color-lighter: #333;
+
+    --sc-tertiary: var(--cm-built-in-color);
+
+    --sc-neutral: #aaa;
+    --sc-neutral-darker: #555;
+    --sc-neutral-lighter: #ccc;
+    --sc-link-color: var(--cm-string-color);
+}
```

Comment:
Would be possible but this would introduce the IDE as an dependency to the docs iirc - which would make headless-builds with docs more difficult.
Also I found it "saner" for me to keep C++ code with file operations to a minimum so it does not crash on some platforms.

And I found that most themes need some slight adjustments, for example `lighter` needs to be either a *lighter* or a *darker* color, based on the brightness of the general theme.
Currently the CSS of the Qt Web Browser does not support color operations which could automate this.

At the same time - setting a custom theme for the docs can still be achieved by over-writing the vars in a `custom.css`.